roles/operator_bundle_scorecard_tests/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Run operator scorecard test"
+  block:
+  - name: "Set facts for the Operator groups"
+    set_fact:
+      OO_BUNDLE: "{{ OO_BUNDLE }}"
```

Comment:
Isn't it already set?

Reply:
`OO_BUNDLE` is set in Ansible, but not as an environment variable

Reply:
This won't set it as an environment variable though.

Reply:
You're right, I thought this comment was on another line.  Updated